src/js/comments.js

```diff
@@ -19,7 +22,22 @@ class Comments {
 	_renderComments() {
 		const scriptElement = document.createElement('script');
 		scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
-		scriptElement.setAttribute('onload', `Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'), {talk: 'https://ft-next-talk-spike.herokuapp.com'})`);
+		scriptElement.setAttribute('onload', `Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'),
```

Comment:
Because you have a ScriptElement JS Object, you can use the property `onload` instead of the DOM attribute, this would be better because it takes a JS function as it's value rather than a string of JS (Your editor will highlight the function, do linting on it etc).

```suggestion
		scriptElement.onload= () => Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'),
```